Azure/sonic-buildimage
# [submodule] Advance sonic-utilities

Signed-off-by: Longxiang Lyu <lolv@microsoft.com>



#### Why I did it
To include `sonic_installer` swap setup changes
```
171eb4f [sonic_installer] Add swap setup support (#1787)
6483b0b QOS fieldvalue refernce ABNF format to string (#1626)
8d16eb5 [Fast-reboot] Set flex counters delay indicator to prevent flex counters enablement after fast-reboot (#1768)
1bc24ca [portstat pfcstat] Unify the packet number format in the output of portstat and pfcstat in all cases (#1755)
d7900dd [config][generic-update] Implementing patch sorting (2nd attempt) (#1794)
7ed9a91 [show] Allow system with no ports in config db run without errors (#1754)
```
#### How I did it
Advance `sonic-utilities` submodule

#### How to verify it
build image and verify the swap could be setup successfully on devices with limited memory resources

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
